pandas/core/indexes/base.py

```diff
@@ -1471,6 +1471,40 @@ def is_object(self):
         return is_object_dtype(self.dtype)
 
     def is_categorical(self):
+        """
+        Check if the index type is categorical.
+
+        The given object must be a pandas dataframe or pandas Series.
+
+        Returns
+        ----------
+        boolean
+            True if index is categorical.
+
+        Examples
+        --------
+        >>> s = pd.Series(["Watermelon","Orange","Apple",
+        ...                 "Watermelon"]).astype('category')
+        >>> df = pd.DataFrame({'Weight (grams)':[2000,230,160,1300]}, index=s)
```

Comment:
spcae after the colon. Space after commas.

Reply:
Thank you for your comments ^^ !! We have already changed it.